src/components/Slider/slider.module.scss

```diff
@@ -0,0 +1,90 @@
+$sliderWidth: 200px;
+
+.sliderContainer {
+    display: block;
+    position: relative;
+    width: $sliderWidth;
+}
+
+.slider {
+    position: relative;
+}
+
+.sliderTrack,
+.sliderRange,
+.sliderValue {
+    position: absolute;
+}
+
+.sliderTrack,
+.sliderRange {
+    border-radius: 3px;
+    height: 5px;
+}
+
+.sliderTrack {
+    background-color: #ced4da;
+    width: 100%;
+    z-index: 1;
+}
+
+.sliderRange {
+    background-color: #9fe5e1;
+    z-index: 2;
+}
+
+.sliderValue {
+    color: #666;
```

Comment:
Take note of other component use of color palette and css vars to enable theming.